packages/node_modules/@node-red/editor-client/src/js/ui/palette.js

```diff
@@ -175,9 +175,19 @@ RED.palette = (function() {
                     $('<button type="button" onclick="RED.workspaces.show(\''+type.substring(8).replace(/'/g,"\\'")+'\'); return false;" class="red-ui-button red-ui-button-small" style="float: right; margin-left: 5px;"><i class="fa fa-pencil"></i></button>').appendTo(popOverContent)
                 }
 
-                var safeType = type.replace(/'/g,"\\'");
+                const safeType = type.replace(/'/g,"\\'");
+                function wrapStr(str) {
+                    if(str.indexOf(' ') >= 0) {
+                        return '"' + str + '"'
+                    }
+                    return str
+                }
```

Comment:
```suggestion
                const wrapStr = function (str) {
                    if(str.indexOf(' ') >= 0) {
                        return '"' + str + '"'
                    }
                    return str
                }
```

Reply:
fix linting "Function declarations should not be placed in blocks. Use a function expression or move the statement to the top of the outer function"